components/form-builder/layout/Publish.tsx

```diff
@@ -0,0 +1,88 @@
+import { useTranslation } from "next-i18next";
+import React, { useCallback } from "react";
+import { CheckCircleOutline } from "@styled-icons/material/CheckCircleOutline";
+import { useAllowPublish } from "../hooks/useAllowPublish";
+import { CancelIcon } from "../icons";
+import styled from "styled-components";
+import { FancyButton } from "../panel/Button";
+import Link from "next/link";
+
+const PrimaryButton = styled(FancyButton)`
+  margin-top: 25px;
+  padding: 10px 15px;
+  background: #26374a;
+  box-shadow: inset 0 -2px 0 #515963;
+  color: white;
+
+  &:hover:not(:disabled),
+  &:active,
+  &:focus {
+    color: #ffffff;
+    background: #1c578a;
+    box-shadow: inset 0 -2px 0 #7a8796;
+  }
+
+  &:hover:active {
+    background: #16446c;
+  }
+`;
+
+export const Publish = () => {
+  const { t } = useTranslation("form-builder");
+  const {
+    data: { title, questions, privacyPolicy, translate, responseDelivery, confirmationMessage },
+    isPublishable,
+  } = useAllowPublish();
+
+  const Icon = ({ checked }: { checked: boolean }) => {
+    return checked ? (
+      <CheckCircleOutline className="w-9 fill-green-700 inline-block" />
+    ) : (
+      <CancelIcon className="w-9 fill-red-700 w-9 h-9 inline-block" />
+    );
+  };
+
+  const handlePublish = useCallback(() => {
+    // call API here
+  }, []);
+
+  return (
+    <>
+      <h1 className="border-0 mb-0">{t("publishYourForm")}</h1>
+      <p>{t("publishYourFormInstructions")}</p>
+
+      <ul className="list-none p-0">
+        <li className="mb-4 mt-8">
+          <Icon checked={title} /> Publisher
+        </li>
```

Comment:
We should probably prepend this with a `visually-hidden` element that says what's going on for assistive technology.

Right now, it will just be read as a list ("Publisher", "Questions", "Privacy statement") but we want to identify which one(s) still need(s) to be filled out for someone using a screen reader to go through the list.

Reply:
Added a `<title>` element and prop to those SVG icons.